src/main/java/uk/gov/justice/hmpps/prison/values/Currency.java

```diff
@@ -0,0 +1,19 @@
+package uk.gov.justice.hmpps.prison.values;
+
+import java.util.Arrays;
+import java.util.Optional;
+
+public enum Currency {
```

Comment:
I can't see it happening anytime soon but if we needed to make the app work using USD someone will update the properties but risk not updating this class.

Reply:
I used Currency to make the transformer code more expressive as I don't like using primitive types to represent domain concepts. Happy to remove it though, just let me know.

Reply:
it tightly couples it's self with the currency config. If someone changed the config to USD your code will just ignore it and default to GBP because your model doesn't support other currencies at the moment. 

Let's keep it simple for now and use primitive types then at least it's constant.

Reply:
if correct TDD practice is followed this is not an issue at all, will remove